game/app/singleton/game_manager.gd

```diff
@@ -0,0 +1,157 @@
+extends Node
+
+var _menu_path = "res://app/scenes/menu.tscn"
+var _games_folder_path = "res://games"
+var _games = []
```

Comment:
why an array? I think the dictionary solution was better

Reply:
Good question, I agree